src/commcare_cloud/ansible/roles/kafka/tasks/main.yml

```diff
@@ -55,6 +55,12 @@
 
 - name: Copies kafka log4j.properties
   copy: src=log4j.properties dest=/opt/kafka/config owner=root group=root mode=644
+  when: kafka_version  == '0.8.2.2'
+  tags: log4j
+
+- name: Copies kafka log4j_new.properties
+  copy: src=log4j_new.properties dest=/opt/kafka/config/log4j.properties owner=root group=root mode=644
```

Comment:
this will need to use the `template` module to resolve the vars

Reply:
no we are not using template for the log4j.properties file